app/src/main/java/fr/free/nrw/commons/network/APIService.kt

```diff
@@ -0,0 +1,23 @@
+package fr.free.nrw.commons.network
+
+import fr.free.nrw.commons.profile.model.AchievementResponse
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+
+interface APIService {
+
+    // https://tools.wmflabs.org/commons-android-app/tool-commons-android-app/uploadsbyuser.py?user=Devanonymous
+    @GET("uploadsbyuser.py")
+    suspend fun getImageUploadCount(
+        @Query("user") username : String
+    ) : Response<Int>
+
+
+    // https://tools.wmflabs.org/commons-android-app/tool-commons-android-app//feedback.py?user=Devanonymous
```

Comment:
In this should I provide kdoc here? and if yes kdoc should I link the common apps API endpoint doc or provide a CURL link here?